internal/workflow/processing_test.go

```diff
@@ -663,21 +666,37 @@ func (s *ProcessingWorkflowTestSuite) TestAMWorkflow() {
 	).Return(nil, nil)
 
 	// Post-preservation activities.
-	s.env.OnActivity(updatePackageLocalActivity, ctx, logger, pkgsvc, mock.AnythingOfType("*workflow.updatePackageLocalActivityParams")).
-		Return(nil, nil).
-		Once()
-	s.env.OnActivity(setStatusLocalActivity, mock.Anything, mock.Anything, mock.Anything, mock.Anything).
-		Return(nil, nil).
-		Never()
-	s.env.OnActivity(completePreservationActionLocalActivity, ctx, pkgsvc, mock.AnythingOfType("*workflow.completePreservationActionLocalActivityParams")).
-		Return(nil, nil).
-		Once()
+	s.env.OnActivity(
+		updatePackageLocalActivity,
+		ctx,
+		logger,
+		pkgsvc,
+		mock.AnythingOfType("*workflow.updatePackageLocalActivityParams"),
+	).Return(nil, nil)
+	// s.env.OnActivity(
+	// 	setStatusLocalActivity,
+	// 	mock.Anything,
+	// 	mock.Anything,
+	// 	mock.Anything,
+	// 	mock.Anything,
+	// ).Return(nil, nil).Never()
```

Comment:
?

Reply:
@jraddaoui oh yeah, I meant to delete that call.  That was at line 669 before I re-wrapped the lines — it wasn't commented, but as far as I can tell it doesn't do anything because of the `Never()` method at the end there. :shrug: